tests/cli/NewFwkLib/PdxTest/PdxTests.cs

```diff
@@ -1582,595 +1582,8 @@ protected void verifySize(IRegion<TKey, TVal> m_region, int expectedSize)
                 throw new Exception("Expected size of " + m_region.FullPath + " to be " +
                    expectedSize + ", but it is " + size);
             }
-            // rjk:need to comment this block
```

Comment:
Making no other changes in this file other than to delete commented out code seems out of scope. Please justify.

Reply:
clean up while hunting for tests using the callback